.github/workflows/ubuntu-noble.yml

```
@@ -90,32 +97,99 @@ jobs:
         apt-get update
         apt-get install -y rocm-dev rocdecode-dev libavformat-dev libavcodec-dev
 
-    - name: Configure
-      timeout-minutes: 30
+    - name: Configure, Build and Test
+      timeout-minutes: 115
       shell: bash
       run: |
         git config --global --add safe.directory ${PWD} &&
         cmake --version
```

```suggestion
        cmake --version
        '${{ matrix.compiler }}' --version
```
We can print the compiler version outside of the if-statement.